bindgen/templates/CallbackHelpers.go

```diff
@@ -0,0 +1,10 @@
+type uniffiCallbackResult C.int32_t
+
+const (
+	idxCallbackFree                                          = 0
```

Comment:
Seems like `idxCallbackFree` should be prefixed with `uniffi` like the rest of the values.

Reply:
fixed